README.md

```diff
@@ -1,7 +1,334 @@
+    bodyMember1,
+    bodyMember2
+};
+
+List<PermissionIdentityBody> membersToAddOrUpdate = new List<PermissionIdentityBody> {
+    new PermissionIdentityBody(new PermissionIdentity(@"acme\team2", PermissionIdentityType.Group)) {
+        Members = new List<PermissionIdentity> {
+            member1,
+            member2
+        }
+    }
+};
+
+// Push user mappings and group members in one batched operation.
+client.PermissionManager.AddOrUpdateIdentities(expandedProviderId, null, mappingsMembersToAddOrUpdate.Concat(membersToAddOrUpdate).ToList());
+```
+
+### Disable a single identity
+You can easily disable an identity. For more information, visit [Disabling a Single Security Identity](https://docs.coveo.com/en/84/cloud-v2-developers/disabling-a-single-security-identity).
+```csharp
+client.PermissionManager.DeleteIdentity(expandedProviderId, new PermissionIdentity(@"acme\johndoe", PermissionIdentityType.User));
+```
+
+### Disable identities in batch
+```csharp
+IList<PermissionIdentity> identitiesToDelete = new List<PermissionIdentity> {
+    new PermissionIdentity(@"acme\member1", PermissionIdentityType.User),
+    new PermissionIdentity(@"acme\member2", PermissionIdentityType.User)
+};
+
+client.PermissionManager.DeleteIdentities(expandedProviderId, null, identitiesToDelete);
+```
+
+### Disable identities older than a specific ordering ID
+Same as the documents, you can disable identities that have an ordering ID smaller than the one you provide. For more information, visit [Disabling Old Security Identities](https://docs.coveo.com/en/33/cloud-v2-developers/disabling-old-security-identities).
+```csharp
+client.PermissionManager.AddOrUpdateIdentity(expandedProviderId, 100, new PermissionIdentityBody(new PermissionIdentity(@"acme\tobedeleted3", PermissionIdentityType.User)));
+client.PermissionManager.AddOrUpdateIdentity(expandedProviderId, 200, new PermissionIdentityBody(new PermissionIdentity(@"acme\tobedeleted4", PermissionIdentityType.User)));
+
+// Wait a little bit, then disable all identities with an ordering ID less than 300.
+client.PermissionManager.DeleteIdentitiesOlderThan(expandedProviderId, 300);
+```
+**Good to know:**
+* As for `DeleteDocumentsOlderThan`, there is a processing delay. However, it is not configurable for this call. For more information about processing delay, visit [QueueDelay](https://docs.coveo.com/en/131/cloud-v2-developers/deleting-old-items-in-a-push-source). 
+
+### Add complex permissions to a document
+The permission model of your system might be more complicated, thus, simple permissions might not be enough to secure your documents. Below is an example of having two levels. One for the `Administrator` of the system and the other one for standard users. For more information, visit [Complex Permission Model Definition Example](https://docs.coveo.com/en/25/cloud-v2-developers/complex-permission-model-definition-example).
```

Comment:
"Below is an example of having two levels."
-> "Below is an example of a two-level permission model."

Reply:
```suggestion
The permission model of your system might be more complicated, thus, simple permissions might not be enough to secure your documents. Below is an example of a two-level permission model. One for the `Administrator` of the system and the other one for standard users. For more information, visit [Complex Permission Model Definition Example](https://docs.coveo.com/en/25/cloud-v2-developers/complex-permission-model-definition-example).
```